test/jobs/retrigger_test.js

```diff
@@ -63,14 +63,24 @@ suite('jobs/retrigger', function() {
         ]);
       });
       test("duplicating a subgraph replaces task IDs referenced in task definition", async function() {
+        // Using a payload more like production task payloads will help ensure that
+        // task Ids nested deeper get replaced.
+        let objectPayload = {
```

Comment:
This is more so to gain confidence that when we have a task ID nested within the payload that it's replaced.  When trying to test retriggering it didn't seem that it was replacing the task ID here, but the tests show that it it is.  Something else must be going wrong but I just wanted to have a test that ensured this.